src/squid.md

```diff
@@ -0,0 +1,280 @@
+  use only.
+
+-O options
+
+> Set Windows Service Command line options in Registry.
+
+-r
+
+> Remove a Windows Service (see `-n` option).
+
+-R
+
+> Do not set **REUSEADDR** on port.
+
+-s
+
+> Enable logging to syslog. Also configurable in
+  **@SYSCONFDIR@/squid.conf**
+
+-S
+
+> Double-check swap during rebuild.
+
+-u port
+
+> Specify ICP port number.
+
+-v
+
+> Print version and build details.
+
+-X
+
+> Force full debugging.
+
+-Y
+
+> Only return `UDP_HIT` or `UDP_MISS_NOFETCH` during fast
+  reload.
+
+-z
+
+> Create missing swap directories and other missing
+  [cache_dir](http://www.squid-cache.org/Doc/config/cache_dir/)
+  structures, then exit.
+
+> All cache_dir types create the configured top-level directory
+  if it is missing. Other actions are type-specific.
+
+> This option does not enable validation of any present swap
+  structures. Its focus is on creation of missing pieces.
+
+> If nothing is missing, squid `-z` just exits.
+
+> If you suspect cache_dir corruption, you must delete the
+  top-level cache_dir directory before running squid `-z`.
+
+> By default, squid `-z` runs in daemon mode so that
+  configuration macros and other SMP features work as expected,
+  returning control to the caller before cache_dirs are fully
+  initialized.
+
+> If run from init scripts or daemon managers, the caller often
+  needs to wait for the initialization to complete before
+  proceeding further.
+
+> Use `-\-foreground` option to prevent premature exits.
+
+> Use `-N` option to disable daemon mode.
+
+# FILES
+
+Squid configuration files located in @SYSCONFDIR@/:
+
+squid.conf
+
+> The main configuration file.
+
+> You must initially make changes to this file for **squid** to
+  work.
+
+squid.conf.default
+
+> Reference copy of the configuration file. Always kept up to
+  date with the version of Squid you are using.
+
+> Use this to look up the default configuration settings and
+  syntax after upgrading.
+
+squid.conf.documented
+
+> Reference copy of the configuration file. Always kept up to
+  date with the version of Squid you are using.
+
+> Use this to read the documentation for configuration options
+  available in your build of Squid.
+
+> The [online configuration manual](http://www.squid-cache.org/Doc/config/)
+  is also available for a full reference of options.
+
+errorpage.css
+
+> CSS Stylesheet to control display of generated error pages.
+
+> Use this to set any company branding you need, it will apply
+  to every language Squid provides error pages for.
+
+Some files also located elsewhere:
+
+mime_table @DEFAULT_MIME_TABLE@
+
+> MIME type mappings for FTP gatewaying
+
+> Can be configured with the
+  [mime_table directive](http://www.squid-cache.org/Doc/config/mime_table/)
+
+error_directory @DEFAULT_ERROR_DIR@
+
+> Location of Squid error pages and templates.
+
+# AUTHOR
+
+Squid was written over many years by a changing team of
+developers and maintained in turn by
```

Comment:
Squid (as a whole) has been maintained by others as well. The list below and "in turn" phrasing are misleading!